scene/gui/link_button.cpp

```diff
@@ -127,6 +127,6 @@ void LinkButton::_bind_methods() {
 
 LinkButton::LinkButton() {
 	underline_mode = UNDERLINE_MODE_ALWAYS;
-	set_enabled_focus_mode(FOCUS_NONE);
+	set_focus_mode(FOCUS_NONE);
```

Comment:
I think the default `focus_mode` is already `FOCUS_NONE`, so this could be removed. Same in `menu_button.cpp`.